### PR TITLE
[DE] Replace english cards with german

### PR DESCRIPTION
**Pull request by DerSchlupf**
Heroes Section:
image

Deckbuilding Section:
image
image

Unit Section:
image

### CODE REVIEW

**Comment by DerSchlupf** [comment]
Does someone has time to find the typo/error for me? :/

File: translations/assets/de/Knowlegde.png
Comment (qwrtln) [inline translations/assets/de/Knowlegde.png:1]: `Konwledge.png` -> `knowledge.png`

**Comment by qwrtln** [comment]
Please show the diff on pictures (before|after) style.

**Comment by qwrtln** [comment]
No, you need to rename the file and commit it.

**Comment by DerSchlupf** [comment]
I know, but its strange. there is nothing to rename: 
image

**Comment by DerSchlupf** [comment]
Yeah it was clear, but it was not the typo you pointed out. It was knowle**gd**e --> knowle**dg**e. Because of that i overlooked the real typo and thought, someone else has renamed it. :)

**Comment by qwrtln** [comment]
> Yeah it was clear, but it was not the typo you pointed out. It was knowle**gd**e --> knowle**dg**e. Because of that i overlooked the real typo and thought, someone else has renamed it. :)

Haha, haven't noticed that. But the name is still wrong, I think? It shouldn't be capitalized.

**Comment by DerSchlupf** [comment]
> Please show the diff on pictures (before|after) style.

Regarding this it could be a problem... I've overwrite the files and havent a copy of the original files (with more MBs). Hmm beginner-fail, sry. If it is necassary i have the first file, the scans without rounded corners. Perhaps we could firstly look at the pics in the PDF, when its build?

> It shouldn't be capitalized.

if i change this, GIT doesnt see any difference. I cant "git add" a changed file. Perhaps its irrelevant?

**Comment by qwrtln** [comment]
> Regarding this it could be a problem... I've overwrite the files and havent a copy of the original files (with more MBs). Hmm beginner-fail, sry. If it is necassary i have the first files, the scans without rounded corners. Perhaps we could firstly look at the pics in the PDF, when its build?

I don't mean with bigger files. With the English ones. We want to be sure that the green circles are in appropriate spots.

**Comment by DerSchlupf** [comment]
Ah okay, done! see first post

**Review by qwrtln — APPROVED** [review]

**Pull request merged**